src/main/kotlin/com/arabyte/arabyteapi/domain/mypage/controller/MyPageController.kt

```diff
@@ -0,0 +1,66 @@
+package com.arabyte.arabyteapi.domain.mypage.controller
+
+import com.arabyte.arabyteapi.domain.article.dto.ArticlePreviewResponse
+import com.arabyte.arabyteapi.domain.mypage.dto.MyPageResponse
+import com.arabyte.arabyteapi.domain.mypage.dto.UpdateBasicInfoRequest
+import com.arabyte.arabyteapi.domain.mypage.dto.UpdateSubInfoRequest
+import com.arabyte.arabyteapi.domain.mypage.enums.MyPageArticleType
+import com.arabyte.arabyteapi.domain.mypage.service.MyPageService
+import com.arabyte.arabyteapi.domain.user.entity.User
+import com.arabyte.arabyteapi.global.annotation.RequestUser
+import io.swagger.v3.oas.annotations.Operation
+import org.springframework.data.domain.Page
+import org.springframework.web.bind.annotation.*
+
+@RestController
+@RequestMapping("/mypage")
+class MyPageController(
+    private val myPageService: MyPageService,
+) {
+    @Operation(
+        summary = "마이페이지 게시물 조회",
+        description = "?type=MY로 요청하면 내가 쓴 게시물, ?type=LIKE로 요청하면 내가 좋아요를 누른 게시물을 반환합니다."
+    )
+    @GetMapping("/articles")
+    fun getMyArticles(
+        @RequestParam type: MyPageArticleType,
+        @RequestUser user: User,
```

Comment:
```suggestion
        @RequestUser user: User,
        @RequestParam type: MyPageArticleType,
```

request user를 항상 맨위에 쓰는걸로하죠

Reply:
반영하겠습니다~